src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -84,6 +85,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $stdout = $output;
```

Comment:
this is not needed. SymfonyStyle also implements OutputInterface. So it can be used just as $stdout.